### DISCUSSION TITLE
I want to over ride Semantic Ui Font family

### DISCUSSION THREAD

**Discussion by VuppalaMahesh** [fomantic/Fomantic-UI#1486] [General] [answered]
# Help Wanted
Hi team,
I see that Semantic Ui used   
     **font-family: Lato,'Helvetica Neue',Arial,Helvetica,sans-serif;**
But in my project I have to use 
**font-family: 'Gothic A1', sans-serif;**

I tried changing in CSS body tag, still  **Helvetica font family** is reflecting in some elements, I have include Gothic A1 family for each element. Please help me how can I make changes so that it will be directly applied to all elements. 
**Note:  I am using Semantic UI CDN links.

## Testcase
I did put little code in below link, you can check removing **.section-alignment .content-title {
font-family { font-family: 'Gothic A1', sans-serif;
}** Help me how to over come with this
https://jsfiddle.net/tpj0uz2n/4/

**Comment by lubber-de** [comment, marked as answer]
Try this

```css
.ui,
.ui *,
body,
h1,
h2,
h3,
h4,
h5 {
  font-family: 'Gothic A1';
}
```
See https://jsfiddle.net/u9o8j6hb/

**Comment by VuppalaMahesh** [comment]
Thank you @lubber-de , it's working! . You people are really awesome! You can close the issue.
